conferences/abstracts/EuroPython2020-talk.md

```diff
@@ -0,0 +1,73 @@
+# EuroPython 2020 Proposal - Talk
+
+**Where:** Online! :woman_technologist:
+
+**When:** 23rd - 26th July 2020
+
+**Conference website:** <https://ep2020.europython.eu/>
+
+**Call for Proposals website:** <https://ep2020.europython.eu/call-for-proposals/>
+
+## Title
+
+Sharing Reproducible Python Environments with Binder
+
+## Subtitle
+
+(_Optional_)
+
+## Abstract (long)
+
+_At most 1500 characters. Current count (with spaces): 1182_
+
+As reproducibility gains traction in the data science community, the need to package code, data and the computational environment is growing.
+
+There are many tools that address different aspects of this type of packaging, such as Jupyter Notebooks for literate programming, Docker for containerising and porting computational environments, and so on.
+But they represent barriers to reproducibility as each one requires time and effort to learn.
+
+Project Binder provides the tooling for generating reproducible computational analyses with a web-based interface, such that users do not have to worry about all the moving parts so long as they have followed software best practices: their code is version controlled and they've captured the dependencies the analysis needs to run.
+Binder then allows the users to easily share their analyses by hosting the compute in the cloud and providing a unique URL to the code repository.
+
+During this talk, Sarah will introduce Binder (the service), BinderHub (the technological infrastructure) and mybinder.org (a public instance of a Binder service, free for anyone to use) and demonstrate how it can be used to share Python environments and analyses.
+
+## Abstract (short)
+
+_At most 500 characters. Current count (with spaces): 441_
+
+Packaging code, data and computational environments to provide reproducible analyses requires extra time and effort to learn the required tools.
+Project Binder provides open-source tooling for executing reproducible analyses in the cloud, and enabling them to be shared via a single, unique URL.
+
+In this talk, Sarah will introduce Binder, BinderHub and mybinder.org and demonstrate how it can be used to share Python environments and analyses.
+
+## Prerequisites for attending the session
+
+(_Optional_)
+
+Familiarity with version control, what a Jupyter Notebook is and an interest in reproducibility.
+
+## Python Skill Level
+
+Beginner
+
+## Domain Expertise
+
+Beginner
+
+## Tags
+
+- Best Practice
+- Data Science
+- Jupyter
+- Open-Source
+- Public Cloud (AWS/Google/...)
+
+## Type
+
+Talk (30 mins)
```

Comment:
What's your Q&A strategy (inline or at the end)? The call makes specific reference to Q&A time

Reply:
Uhh, my Q&A strategy is usually wing it 😬 Sorry, converting the form into Markdown hasn't made this very clear but this is a dropdown box selection.